pages/_document.js

```diff
@@ -11,11 +11,28 @@ class MyDocument extends Document {
     return (
       <Html>
         <Head>
-          <title>@SocialGouv/carnets</title>
           <link
             rel="stylesheet"
+            integrity="rKNSttpemxgXPOY2DHw/JxXSzQzBGnL4XNK2daofRRgeHB7GL77kpu+0FYr+v1Cc"
             href="https://socialgouv.github.io/bootstrap/master/@socialgouv/bootstrap.core/dist/socialgouv-bootstrap.min.css"
           />
+          <script
+            src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+            integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
+            crossOrigin="anonymous"
+          ></script>
+
+          <script
+            src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
+            integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
+            crossOrigin="anonymous"
+          ></script>
+
+          <script
+            src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
+            integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
+            crossOrigin="anonymous"
+          ></script>
```

Comment:
not sure we need all of this... 3rd party JS doesnt play well with React

in last resort, these should be `required` at node level so we dont need to request other servers and leak cookies/tracking info

Reply:
Okay, understood!